Source/ACE/Entity/Landblock.cs

```diff
@@ -596,6 +603,18 @@ private void HandleGameAction(QueuedGameAction action, Player player)
                         HandleSoundEvent(obj, soundEffect);
                         break;
                     }
+                case GameActionType.IdentifyObject:
+                    {
+                        var g = new ObjectGuid(action.ObjectId);

```

Comment:
we know the retail servers throttled this, we might want to put a "TODO" comment here to that effect.